tests/msim/test_msim.cpp

```diff
@@ -59,6 +57,36 @@ void pressure(const EclipseState& es, const Schedule& /* sched */, data::Solutio
     std::fill(data.begin(), data.end(), units.to_si(UnitSystem::measure::pressure, seconds_elapsed));
 }
 
+bool is_file(const boost::filesystem::path& name)
+{
+    return boost::filesystem::exists(name)
+        && boost::filesystem::is_regular_file(name);
+}
+
+class WrkArea
```

Comment:
This should in my opinion be extracted to some test-lib.

Reply:
Sure, we can do that.